pdfjs.config

```diff
@@ -1,5 +1,5 @@
 {
-  "stableVersion": "3.10.111",
-  "baseVersion": "e142baecb15dd84d6058e217f227a4a188209fe4",
-  "versionPrefix": "3.11."
+  "stableVersion": "3.11.174",
+  "baseVersion": "ce87167432819f85df49b6b16c7a78556e9a4ee0",
+  "versionPrefix": "3.12."
```

Comment:
Is there a reason for this, since there's nothing tagged `api-minor` in the last release?

Reply:
Whoops; fixed now. I was so used to us always having at least one `[api-minor]` change that I missed that we had none this time. Good catch!

Reply:
Actually, don't we need to keep the `baseVersion` as well to prevent https://github.com/mozilla/pdf.js/blob/master/gulpfile.mjs#L757 from messing up the build-number?

Reply:
Hm, yes, I also think it'll otherwise start at patch version 0 again.